samples/web/content/apprtc/js/appwindow.js

```diff
@@ -61,10 +61,12 @@ var params = {
 // Generate random room id and connect.
 params.roomId = randomString(9);
 params.roomLink =  'https://apprtc.appspot.com/room/' + params.roomId;
-params.roomServer = 'https://apprtc.appspot.com';
+params.server = 'https://apprtc.appspot.com';
```

Comment:
Had to line up with this change from the refactoring:
+var Call = function(params) {
+  this.params_ = params;
+  this.roomServer_ = params.server || '';

I could change it to roomServer in the other place if you prefer?